src/group_theory/exponent.lean

```diff
@@ -7,13 +7,14 @@ import group_theory.order_of_element
 import algebra.punit_instances
 import algebra.gcd_monoid.finset
 import ring_theory.int.basic
+import group_theory.specific_groups.cyclic
 
 /-!
 # Exponent of a group
 
 This file defines the exponent of a group, or more generally a monoid. For a group `G` it is defined
-to be the minimal `n≥1` such that `g ^ n = 1` for all `g ∈ G`. For a finite group `G` it is equal to
-the lowest common multiple of the order of all elements of the group `G`.
+to be the minimal `n≥1` such that `g ^ n = 1` for all `g ∈ G`. For a finite left cancel monoid
```

Comment:
I think that `exponent` is defined for left cancellative monoids is an implementation detail which can be mentioned in brackets or in a section about implementation details, but doesn't deserve to be mentioned in the summary.